iroh/src/discovery/pkarr.rs

```diff
@@ -236,8 +227,8 @@ struct PublisherService {
 impl PublisherService {
     async fn run(mut self) {
         let mut failed_attempts = 0;
-        let republish = tokio::time::sleep(Duration::MAX);
-        tokio::pin!(republish);
+        let republish = time::sleep(Duration::MAX);
+        n0_future::pin!(republish);
```

Comment:
nit: the pin macro exists in the std lib

Reply:
TIL

Reply:
Seems like it doesn't work though. If I use `std::pin::pin!(republish)` it doesn't compile for me :thinking: